Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM gehlenborglab/higlass
+FROM gehlenborglab/higlass:v0.2.61
 
 COPY on_startup.py /home/higlass/projects/higlass-server
-COPY input.json $HIGLASS_SERVER_BASE_DIR
 
-# Display our waiting page until higlass ingests all tilesets
-COPY index.html /home/higlass/projects/higlass-website/index.html
+
+# Swap the "app" html with the main html to always provide the /app/ view
+RUN cp /home/higlass/projects/higlass-website/app/index.html /home/higlass/projects/higlass-website/index.html
```

Comment:
Would you just want to do a batch copy and maybe get the static assets, etc?

Reply:
Oh, this is a hack to only allow the HiGlass `/app` view i.e. the fullscreen mode. Just overwriting the main `index.html`, static assets are already in the proper places at this point